### PR TITLE
Update README link in development.md

### PR DESCRIPTION
**Pull request by Harshdev098**
### Why:
a link in READMEs section of development.md is not working

Closes: 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by DanyRaby5008** [comment]
Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
From: Harsh Dev Pathak ***@***.***>
Sent: Saturday, October 14, 2023 11:32:06 PM
To: github/docs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [github/docs] Update README link in development.md (PR #29166)

Why:a link in READMEs section of development.md is not working

Closes:

What's being changed (if available, include any code snippets, screenshots, or gifs):
Check off the following:

  *    I have reviewed my changes in staging, available via the View deployment link in this PR's timeline.

     *   For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the data directory.
  *    For content changes, I have completed the self-review checklist<https://github.com/github/docs/blob/main/contributing/self-review.md#self-review>.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/github/docs/pull/29166

Commit Summary

  *   b89a908<https://github.com/github/docs/pull/29166/commits/b89a908a42b056ae2f8625e9d1ef911b3e253f40> Update README link in development.md

File Changes

(1 file<https://github.com/github/docs/pull/29166/files>)

  *   M contributing/development.md<https://github.com/github/docs/pull/29166/files#diff-72bcb91767571334c253e34f35208a6a98c3a397e444f1e638f4cf18d11df9e8> (2)

Patch Links:

  *   https://github.com/github/docs/pull/29166.patch
  *   https://github.com/github/docs/pull/29166.diff

File: contributing/development.md [hunk 1]
```diff
@@ -74,5 +74,5 @@ For more info about working with this site, check out these READMEs:
 <!-- The following will be moved into src directory over time: -->
 - [components/README.md](../components/README.md)
 - [middleware/README.md](../middleware/README.md)
-- [script/README.md](../script/README.md)
+- [script/README.md](script/bookmarklets/README.md)
 - [tests/README.md](../tests/README.md)
```
Comment (jhosman) [inline contributing/development.md:78]:
```suggestion
- [src/README.md](src/README.md)
```

File: contributing/development.md [hunk 2]
```diff
@@ -74,5 +74,5 @@ For more info about working with this site, check out these READMEs:
 <!-- The following will be moved into src directory over time: -->
 - [components/README.md](../components/README.md)
 - [middleware/README.md](../middleware/README.md)
```
Comment (jhosman) [inline contributing/development.md:76]:
```suggestion
```

**Review by jhosman — COMMENTED** [review]
Thanks for spotting this and for opening a PR with a fix! 

I’ve made an additional update, so that it links to the new location in the `src` directory. (You can see the [commit a few months ago](https://github.com/github/docs/commit/23a0b7d047c1f37e33a850806558cef0a15a047d) where we fixed this in the main README file, but we forgot to make those same changes in `development.md`.) 

We'll go ahead and get this merged for you once tests are passing! 🚀

**Review by jhosman — APPROVED** [review]

**Pull request merged**